pkg/kubelet/network/cni/cni.go

```diff
@@ -189,6 +183,14 @@ func (plugin *cniNetworkPlugin) Name() string {
 	return CNIPluginName
 }
 
+func (plugin *cniNetworkPlugin) Status() error {
```

Comment:
@mikedanese every 30s from kubelet's node sync loop.  But if that's getting called periodically, then we don't need the 10s forever loop.  Why have two loops when one will do?